values.yaml

```diff
@@ -98,13 +98,16 @@ georchestra:
       #registry_secret: default
       envsubst:
         enabled: true
+    gateway:
+      enabled: true
+      docker_image: georchestra/gateway:latest
   datadir:
     volume:
     - name: georchestra-datadir
       emptyDir: {}
     git:
       url: https://github.com/georchestra/datadir.git
-      ref: k8s-envs-master
+      ref: k8s-master-gateway
```

Comment:
do note that we will need to use the correct branch before merging this PR: `k8s-master`